Documentation/install/guides/kops.rst

```diff
@@ -4,37 +4,167 @@
 
-.. _kops: https://github.com/kubernetes/kops
-.. _ami: https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/AMIs.html
+Installing kops
+===============
 
 .. code:: bash
 
-    $ aws ec2 describe-images --region=us-east-1 --owner=595879546273 \
-        --filters "Name=virtualization-type,Values=hvm" "Name=name,Values=CoreOS-stable*" \
-        --query 'sort_by(Images,&CreationDate)[-1].{id:ImageLocation}'
+        curl -LO https://github.com/kubernetes/kops/releases/download/$(curl -s https://api.github.com/repos/kubernetes/kops/releases/latest | grep tag_name | cut -d '"' -f 4)/kops-linux-amd64
+        chmod +x kops-linux-amd64
+        sudo mv kops-linux-amd64 /usr/local/bin/kops
 
-    {
-        "id": "595879546273/CoreOS-stable-1576.4.0-hvm"
-    }
+        # If you are on MacOS, you can use:
+        brew update && brew install kops
 
-You also need to change the default etcd version used in kops, as Cilium needs at least version 3.1 (kops default is still on 2.0 branch).
 
-The following is an example command for creating a Cilium-backed kops cluster:
+
+Setting up IAM Group and User 
+=============================
+
+Assuming you have all the pre-requisites, run the following commands to create the ``kops`` user and group
+
+.. code:: bash
+
+        # Create IAM group named kops and grant access
+        aws iam create-group --group-name kops
+        aws iam attach-group-policy --policy-arn arn:aws:iam::aws:policy/AmazonEC2FullAccess --group-name kops
+        aws iam attach-group-policy --policy-arn arn:aws:iam::aws:policy/AmazonRoute53FullAccess --group-name kops
+        aws iam attach-group-policy --policy-arn arn:aws:iam::aws:policy/AmazonS3FullAccess --group-name kops
+        aws iam attach-group-policy --policy-arn arn:aws:iam::aws:policy/IAMFullAccess --group-name kops
+        aws iam attach-group-policy --policy-arn arn:aws:iam::aws:policy/AmazonVPCFullAccess --group-name kops
+        aws iam create-user --user-name kops
+        aws iam add-user-to-group --user-name kops --group-name kops
+        aws iam create-access-key --user-name kops
+
+
+kops requires a creation of a dedicated s3 bucket in order to store the state and representation of the cluster. You will need to change the bucket name and provide your unique bucket name (for example a reverse of FQDN added with short description of the cluster). Also make sure to use the region where you will be deploying the cluster. 
+
+.. code:: bash
+
+        aws s3api create-bucket --bucket prefix-example-com-state-store --region us-west-2 --create-bucket-configuration LocationConstraint=us-west-2
+        export KOPS_STATE_STORE=s3://prefix-example-com-state-store
+
+Above steps are sufficient for getting a working cluster installed. Please consult `kops aws documentation <https://github.com/kubernetes/kops/blob/master/docs/aws.md>`_ for more detailed setup
+
+
+Cilium Prerequisites
+====================
+
+* Cilium requires minimum kernel version to be 4.8
```

Comment:
Instead of duplicating these requirements, I'd prefer linking to to the minimum system requirements as outlined in `Documentation/install/system_requirements.rst` so that we only have to maintain one source of truth for such requirements.